models/stg_zendesk__ticket.sql

```diff
@@ -21,6 +21,13 @@ fields as (
                 staging_columns=get_ticket_columns()
             )
         }}
+
+        --The below script allows for pass through columns.
+        {% if var('zendesk__ticket_passthrough_columns',[]) %}
```

Comment:
I can confirm that this line indeed does work! However, I still can't seem to wrap my mind around this jinja notation. I tried a couple of different other approaches and there are several ways to do this, I think the below may be easiest to understand while still only using one condition. What do you think about using something more intuitive to read like:
`if var('zendesk__ticket_passthrough_columns')|length > 0` instead?